docs/maps/search.asciidoc

```diff
@@ -49,13 +49,15 @@ Spatial filters have the following properties:
 * *Geometry label* enables you to provide a meaningful name for your spatial filter.
 * *Spatial field* specifies the geo_point or geo_shape field used to determine if a document matches the spatial relation with the specified geometry.
 * *Spatial relation* determines the {ref}/query-dsl-geo-shape-query.html#_spatial_relations[spatial relation operator] to use at search time. Only available when *Spatial field* is set to geo_shape.
+* *Action* specifies where the filter will be applied. Select *Apply filter to current view* to apply the filter to the current view. Select a drilldown action to apply the filter to the drilldown action. Only available when the map is a panel in a {kibana-ref}/dashboard.html[dashboard] with {kibana-ref}/drilldowns.html[drilldowns].
```

Comment:
There's a lot of repetition here.  Maybe something like this:

Action specifies whether to apply the filter to current view to or to a drilldown action. This property is only available when the map...